Makefile

```diff
@@ -13,14 +13,18 @@
 # file that was distributed with this source code.                         #
 ############################################################################
 
+ifeq ("symfony-cmf/content-bundle", "symfony-cmf/testing")
```

Comment:
could we do this `if` when generating the makefile instead, so that the file does not contain references to random packages when put into other packages?

Reply:
that one was really tricky, but yes i will try to do it in an other way.